compiler/src/dmd/dsymbol.d

```diff
@@ -851,7 +851,7 @@ extern (C++) class Dsymbol : ASTNode
      */
     extern (D) static bool oneMembers(Dsymbols* members, out Dsymbol ps, Identifier ident)
     {
-        //printf("Dsymbol::oneMembers() %d\n", members ? members.length : 0);
+        printf("Dsymbol::oneMembers() %d\n", members ? cast(int) members.length : 0);
```

Comment:
don't forget to turn this `printf` back off.

Reply:
I really should commit before constantly switching branches :sweat_smile: